app/controllers/picks_controller.rb

```diff
@@ -44,16 +42,15 @@ def destroy
   end
 
   def previous
-    @picks = current_user
-              .picks
-              .includes([:game => [:home_team, :away_team]])
+    @picks = current_user.picks.includes([:game => [:home_team, :away_team]])
     @weeks = (1...current_week+1).to_a.reverse
     render "index"
   end
 
   def standings
-    @users = User.where.not(id: [64, 65, 66]).all.to_a.sort_by(&:points).
-    reverse.to_json(:methods => [:wins, :losses, :pushes, :percent, :points])
+    @current_week = current_week
+    @users = User.where.not(id: [64, 65, 66]).all.to_a
+      .to_json(:methods => [:wins, :losses, :pushes, :percent, :points, :week_standings])
```

Comment:
this is where we actually boostrap the `week_standings` to the frontend. All the methods we include here will run on the backend and be injected onto the frontend.